[2025-02-05T06:37:41.500+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:37:41.550+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: task_group_example.data_processing.transform_tasks.transform_a scheduled__2024-02-09T00:00:00+00:00 [queued]>
[2025-02-05T06:37:41.553+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: task_group_example.data_processing.transform_tasks.transform_a scheduled__2024-02-09T00:00:00+00:00 [queued]>
[2025-02-05T06:37:41.553+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-02-05T06:37:41.557+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): data_processing.transform_tasks.transform_a> on 2024-02-09 00:00:00+00:00
[2025-02-05T06:37:41.559+0000] {standard_task_runner.py:57} INFO - Started process 1063 to run task
[2025-02-05T06:37:41.560+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'task_group_example', 'data_processing.transform_tasks.transform_a', 'scheduled__2024-02-09T00:00:00+00:00', '--job-id', '129', '--raw', '--subdir', 'DAGS_FOLDER/task_group_dag.py', '--cfg-path', '/tmp/tmp078a7t3_']
[2025-02-05T06:37:41.561+0000] {standard_task_runner.py:85} INFO - Job 129: Subtask data_processing.transform_tasks.transform_a
[2025-02-05T06:37:41.567+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:37:41.579+0000] {task_command.py:415} INFO - Running <TaskInstance: task_group_example.data_processing.transform_tasks.transform_a scheduled__2024-02-09T00:00:00+00:00 [running]> on host 3849bfb82152
[2025-02-05T06:37:41.583+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:37:41.606+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='task_group_example' AIRFLOW_CTX_TASK_ID='data_processing.transform_tasks.transform_a' AIRFLOW_CTX_EXECUTION_DATE='2024-02-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-09T00:00:00+00:00'
[2025-02-05T06:37:41.607+0000] {logging_mixin.py:151} INFO - 执行任务: 转换A
[2025-02-05T06:37:41.607+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-05T06:37:41.609+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=task_group_example, task_id=data_processing.transform_tasks.transform_a, execution_date=20240209T000000, start_date=20250205T063741, end_date=20250205T063741
[2025-02-05T06:37:41.618+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-05T06:37:41.627+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
